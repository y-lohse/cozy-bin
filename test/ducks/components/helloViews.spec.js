'use strict'

/* eslint-env jest */

import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import { Hello1 } from '../../../src/components/HelloViews/Hello1'
import { Hello2 } from '../../../src/components/HelloViews/Hello2'
import { Hello3 } from '../../../src/components/HelloViews/Hello3'

Enzyme.configure({ adapter: new Adapter() })

describe('Hello components:', () => {
  it('Hello1 should be rendered correctly', () => {
    const component = shallow(<Hello1 />).getElement()
    expect(component).toMatchSnapshot()
  })
  it('Hello2 should be rendered correctly', () => {
    const component = shallow(<Hello2 />).getElement()
    expect(component).toMatchSnapshot()
  })
  it('Hello3 should be rendered correctly', () => {
    const component = shallow(<Hello3 />).getElement()
    expect(component).toMatchSnapshot()
  })
})
