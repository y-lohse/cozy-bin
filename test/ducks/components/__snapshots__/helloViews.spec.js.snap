// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello components: Hello1 should be rendered correctly 1`] = `
<p
  style="margin-left: 2em"
>
  Just... Hello world! This is View 1
</p>
`;

exports[`Hello components: Hello2 should be rendered correctly 1`] = `
<p
  style="margin-left: 2em"
>
  Just... Hello world! This is View 2
</p>
`;

exports[`Hello components: Hello3 should be rendered correctly 1`] = `
<p
  style="margin-left: 2em"
>
  Just... Hello world! This is View 3
</p>
`;
